app/src/main/res/layout/activity_about.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!--
-  Copyright (C) 2018 Axel Paetzold
+  Copyright (C) 2020 Axel Paetzold
```

Comment:
The original copyright year should still be visible. Use 2018, 2020 or for consecutive years e.g. 2019-2020.

Reply:
2020 is original. 2018 was a copy-paste error